compose/.run/androidx.benchmark.integration-tests.macrobenchmark-target.run.xml

```diff
@@ -0,0 +1,60 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
These configurations are predefined in Android Studio, but we hide them under the directory `android`

Reply:
What's the point of having those under VC?

Reply:
Are you asking why even have the desktop configurations under VC?

They hard to find in the usual Gradle tasks.

When we develop in IDE - it is useful to have the most popular tasks in IDEA run configurations